tax_credit/tests/test_simulated_communities.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2014--, tax-credit development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+from unittest import TestCase, main
+from shutil import rmtree
+from os import makedirs
+from os.path import join, exists
+from tax_credit.simulated_communities import (generate_simulated_communities)
+
+
+class EvalFrameworkTests(TestCase):
+
+    def setUp(self):
+        self.table2 = table2.split('\n')
+
+    def test_generate_simulated_communities(self):
+        testdir = 'sim_test'
+        comm_dir = 'blob'
+        testpath=join(testdir, comm_dir)
+        if not exists(testpath):
+            makedirs(testpath)
```

Comment:
As above, would recommend making a temporary directory in a more portable way using `tempfile.mkdtemp`.